crepr/crepr.py

```diff
@@ -377,5 +377,32 @@ def remove(
                 f.write("\n".join(src))
 
 
+@app.command()
+def report_missing(
+    files: Annotated[list[pathlib.Path], file_arg],
+) -> None:
+    """Report classes without __repr__ methods."""
+    for file_path in files:
```

Comment:
```suggestion
    for module, file_path in get_modules(files):
```
I don't think we need to reinvent the wheel here